test/test_default_html.py

```diff
@@ -29,6 +29,14 @@
                 (".//section[@id='positional-arguments']", ''),
                 (".//section[@id='positional-arguments']/dl/dt[1]/kbd", 'baz'),
             ],
+            'special-characters.html': [
+                (".//h1", 'Sample', False),
+                (".//h1", 'Special Characters'),
+                (".//section/dl/dd/p", 'Default:'),
+                (".//section/dl/dd/p/code/span", '420'),
+                (".//section/dl/dd/p/code/span", "'*.rst"),
+                (".//section/dl/dd/p/code/span", r"\['\*.rst',"),
```

Comment:
What does this look like in the rendered HTML (both HTML source, and in a browser) please?

Reply:
The result from the test method:
![image](https://github.com/sphinx-doc/sphinx-argparse/assets/12717020/6d57dda0-0411-4648-acf8-965aff0f3ba9)

In HTML (formatting mine):
```
<p>
Default:
<code class="docutils literal notranslate">
<span class="pre">'*.rst</span> 
<span class="pre">_txt_</span> 
<span class="pre">**strong**</span> 
<span class="pre">*italic*</span> 
<span class="pre">\`\`code\`\`'</span>
</code>
</p>
```